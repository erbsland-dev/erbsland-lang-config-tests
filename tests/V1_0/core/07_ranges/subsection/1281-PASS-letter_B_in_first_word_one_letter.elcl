# Erbsland Configuration Language Test File
[main.B_name]