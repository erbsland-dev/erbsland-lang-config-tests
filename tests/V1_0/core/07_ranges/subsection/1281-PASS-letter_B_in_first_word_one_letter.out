main = IntermediateSection()
main.b_name = SectionWithNames()
